### PR TITLE
[WIP] Move track icons to track repos

### PR DESCRIPTION
**Pull request by joeltaylor**
WIP

Fixes:  #2925

### CODE REVIEW

**Comment by joeltaylor** [comment]
@kytrinyx This PR is good to go. I ended up fetching the track within the image helper...too much?

**Comment by kytrinyx** [comment]
No, I think that's fine for now. We should check that it gets cached, probably, (I don't recall if I added anything for that).

I'm going to leave this PR open until all of the tracks have copied their icons to the various repositories.

File: app/helpers/track_image.rb [hunk 1]
```diff
@@ -2,15 +2,15 @@ module ExercismWeb
...
-        img = "/img/tracks/%s.png" % track_id
-        if File.exist?("./public/%s" % img)
-          img
+      def track_image(track_id)
+        track = ::X::Track.find(track_id)
+        if track.image
+          track.image
         else
           "/img/e_red.png"
         end
```
Comment (tejasbubane) [inline app/helpers/track_image.rb]: `track.image || "/img/e_red.png"`

Comment (joeltaylor) [inline app/helpers/track_image.rb]: Oh, wow. That was a sloppy conditional on my part. Great catch, thank you!

Will change that depending on what gets decided in https://github.com/exercism/x-api/pull/112. We might serve the default image from x-api, which would simplify that method to just `track.image`.

**Comment by kytrinyx** [comment]
We finished this yesterday.

I ended up implementing the track images using the gem that I extracted from x-api (merged in 2608bcc5da900d7897690666c0ab3c736a8067f1).

Thanks your your help along the way!

**Pull request closed without merging**